services/core-api/app/api/mines/incidents/resources/mine_incidents.py

```diff
@@ -106,8 +105,7 @@ def post(self, mine_guid):
                         'Dangerous occurrences require one or more cited sections of HSRC code 1.7.3'
                     )
 
-        reported_timestamp_default = datetime.utcnow(
-        ) if not data['reported_timestamp'] and is_minespace_user() else data['reported_timestamp']
+        reported_timestamp_default = data.get('reported_timestamp', datetime.utcnow())
```

Comment:
I was wrong about this when we spoke about it before, you were doing this the right way the first time: `reported_timestamp_default = datetime.utcnow(
        ) if not data['reported_timestamp'] else data['reported_timestamp']` I think should be fine.

Reply:
merged yours in and dealt with this.